README.md

```diff
@@ -235,19 +235,39 @@ written as a JavaScript string between the name and separating equality sign.
 Rules need to be separated only by whitespace (their beginning is easily
 recognizable), but a semicolon (“;”) after the parsing expression is allowed.
 
-The first rule can be preceded by an _initializer_ — a piece of JavaScript code
-in curly braces (“{” and “}”). This code is executed before the generated parser
-starts parsing. All variables and functions defined in the initializer are
-accessible in rule actions and semantic predicates. The code inside the
-initializer can access options passed to the parser using the `options`
-variable. Curly braces in the initializer code must be balanced. Let's look at
-the example grammar from above using a simple initializer.
+The first rule can be preceded by a _top level initializer_ and/or a _parse
+level initializer_, in that order. Both are pieces of JavaScript code in double
+curly braces (“{{” and “}}”) and single curly braces (“{” and “}”) respectively.
+All variables and functions defined in both _initializers_ are accessible in
+rule actions and semantic predicates. Curly braces in both _initializers_ code
+must be balanced.
+
+The _top level initializer_ is executed once and only once, when the generated
+parser is loaded (through a `require` or an `import` statement for instance). It
+is the ideal location to require, to import or to declare utility functions to
+be used in rule actions and semantic predicates.
+
+The _parse level initializer_ is called everytime and before the generated
```

Comment:
typo?

Reply:
I wanted to emphasize the fact the initializer is called repeatedly but I can live without the "everytime and" ;)